### DISCUSSION TITLE
Strongbox Icon Request

### DISCUSSION THREAD

**Discussion by arifmetic** [elrumo/macOS_Big_Sur_icons_replacements#1470] [Icon request]
Would someone be able to do a Big Sur style icon with skeuomorphic elements for [Strongbox](https://strongboxsafe.com) app? Greatly appreciated.

**Comment by vikasbhatia** [comment]
Something like this?
<img width="256" alt="Strongbox" src="https://github.com/elrumo/macOS_Big_Sur_icons_replacements/assets/11446387/9150bdfe-76ed-483a-80d7-2dbe69300e21">
If you like it, you'll be able to download it from my profile once it's approved: https://macosicons.com/#/u/sidewinder

**Reply by jadtl** [reply]
Hey, could you send it to me? The one of the website returns an access denied error to me :(

**Reply by vikasbhatia** [reply]
Hey @jadtl of course! Sorry I didn't see your comment earlier.

Here you go: https://we.tl/t-3B8lmG66MY

By the way, the PNG also works in a pinch, you don't _need_ the ICNS file per se to swap out the icon. If you drag and drop the PNG onto the existing icon in the Get Info window (or better, use [Pictogram](https://pictogramapp.com/)) then it works perfectly fine with the PNG as well, which you can right-click and save from above :)

PS. I didn't have the ICNS downloaded from earlier, but I converted the PNG to Apple's ICNS format using [this script](https://github.com/BenSouchet/png-to-icns). Maybe you'll find it useful too.

**Reply by jadtl** [reply]
Thanks a ton! I'm familiar with the ICNS converter program used in the script, thanks for pointing that PNG also works fine :)
